src/status_im2/contexts/chat/messages/view.cljs

```diff
@@ -18,8 +18,9 @@
       :chat           chat
       :header-comp    (fn [{:keys [scroll-y]}]
                         [messages.navigation/navigation-view {:scroll-y scroll-y}])
-      :footer-comp    (fn [{:keys [insets]}]
-                        (if-not able-to-send-message?
-                          [contact-requests.bottom-drawer/view chat-id contact-request-state
-                           group-chat]
-                          [:f> composer/composer insets]))}]))
+      :footer-comp    (when (some? able-to-send-message?)
```

Comment:
hm, isn't `able-to-send-message?` a boolean value? in that case it might be renamed as well

Reply:
yes, its boolean value but it can be true/false and nil (means there is no chat in [:db :chats] with chat-id).

Reply:
replaced with `(when chat`